packages/lib-classifier/package.json

```diff
@@ -47,7 +47,7 @@
     "react": "~16.13.x",
     "react-dom": "~16.13.x",
     "seven-ten": "~3.4.x",
-    "styled-components": "~4.1.x"
+    "styled-components": "~5.x.x"
```

Comment:
Any reason this differs from the `~5.1.1` definition in the `app-content-pages`?

Reply:
I don't want the peer dependency requirement to be locked to a specific patch version.